api/envoy/api/v2/rds.proto

```diff
@@ -44,7 +45,18 @@ service RouteDiscoveryService {
   }
 }
 
-// [#comment:next free field: 9]
+// * HTTP :ref:`router filter <config_http_filters_router>`
+// The resource_names_subscribe field in DeltaDiscoveryRequest contains a list of virtual host names or aliases
+// to track. The contents of an alias would be the contents of a "host" or "authority" header used to make an
+// http request. xDS server will match an alias to a virtual host based on the content of VirtualHosts' "domains"
+// field. The resource_names_unsubscribe field contains a list of virtual host names that have been dropped from
```

Comment:
This should reference https://github.com/envoyproxy/envoy/blob/master/api/XDS_PROTOCOL.md#unsubscribing-from-resources

Reply:
Added the reference